scripts/find-testimonials.py

```
@@ -0,0 +1,791 @@
+            skipped_long += 1
+            continue
+            
+        # Skip testimonials that are just code blocks or links
+        if testimonial["body"].strip().startswith("```") and testimonial["body"].strip().endswith("```"):
+            skipped_code += 1
+            continue
+            
+        # Skip testimonials that are likely just markdown formatting or HTML
+        body = testimonial["body"].strip()
+        if (body.startswith("<!--") or 
+            (body.startswith("<") and body.endswith(">") and not "<img" in body) or
```

Ruff (0.11.9)

Test for membership should be `not in`

Convert to `not in`

(E713)